js/ElementRendererProvider.js

```diff
@@ -358,9 +359,18 @@ export default class ElementRendererProvider {
         divEl.style.cssText = Utils.styleToCss(config.style);
       }
 
-      if (config.accessibility && config.accessibility.web) {
+      if (accessibilityWeb) {
         Utils.appendAttributesFromObject(divEl, config.accessibility.web);
       }
+      if ((accessibilityWeb && !accessibilityWeb.tabindex) || !config.accessibility) {
+        divEl.setAttribute('tabindex', '0');
+        divEl.onkeydown = (event) => {
```

Comment:
@FeelsGOOD I think keydown listener should be moved out of the `if` statement. It should be added anyway regardless from we have something in `accessibility.web` or not.